modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxManager.java

```diff
@@ -796,6 +796,16 @@ public GridNearTxLocal threadLocalTx(GridCacheContext cctx) {
         return null;
     }
 
+    /**
+     * Sets transaction for current thread.
```

Comment:
It is absolutely not an obvious thing, that the `null` value means `remove`. Could you please improve javadoc?